doc/commands/offline.8.rst

```diff
@@ -0,0 +1,87 @@
+..
+    Copyright Contributors to the libdnf project.
+
+    This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+    Libdnf is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    Libdnf is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+
+.. _offline_command_ref-label:
+
+################
+ Offline Command
+################
+
+Synopsis
+========
+
+``dnf5 offline <subcommand> [options]``
+
+
+Description
+===========
+
+The ``offline`` command is used to manage "offline" transactions---transactions that run when the system is booted into a minimal environment. Running a transaction in this stripped-down environment can be safer than running it when the system is booted normally since the transaction is less likely to interfere with running processes.
```

Comment:
Is the `transactions---transactions` a typo?

Reply:
That was intended, RST compiles it to an em dash:

> The `offline` command is used to manage “offline” transactions—transactions that run when the system is booted into a minimal environment.